CHANGELOG.md

```diff
@@ -1,6 +1,66 @@
 kOS Mod Changelog
 =================
 
+# v1.0.1 (for KSP 1.1.3) Let's take some input!
+
```

Comment:
I think it would be a good idea to add a sentence or two here at the top indicating the reason why this is 1.1 and not 1.2.  (for the sake of people who won't or can't update yet especially because Remote Tech won't be up to 1.2 for a little while yet.).  The reason this is important is that it also has the side effect of calling out the fact that this is NOT the 1.2 release.  Given the timing of it, lots of people are going to automatically make that assumption and not notice the little 1.1.3 above.